model_zoo/census_wide_deep_model/wide_deep_functional_api.py

```diff
@@ -0,0 +1,179 @@
+import tensorflow as tf
+
+from model_zoo.census_wide_deep_model.feature_config import (
+    FEATURE_COLUMNS,
+    FEATURE_GROUPS,
+    MODEL_INPUTS,
+    TransformOp,
+    get_id_boundaries,
+)
+from model_zoo.census_wide_deep_model.keras_process_layer import (
+    AddIdOffset,
+    CategoryHash,
+    CategoryLookup,
+    NumericBucket,
+)
+
+
+def custom_model():
```

Comment:
It would be better to mark which part is user developed in model zoo, which part is auto-generated from code_gen.